homeassistant/components/device_tracker/bluetooth_tracker.py

```diff
@@ -79,7 +80,13 @@ def discover_devices():
 
     request_rssi = config.get(CONF_REQUEST_RSSI, False)
 
-    def update_bluetooth(now):
+    def update_bluetooth():
```

Comment:
I thought the `now` when calling the function from `track_point_in_utc_time`  ?

Reply:
Look at the changeset, '''track_point_in_utc_time''' is called with '''dt_util.utcnow()''' not with '''now''', I assumed that the unused now was not being used due to the delay introduced by it being called at the end of the function which means that time would be in the past when it was called, which could cause it to just continuously run again after the interval + time it starts, rather than interval+ time it completed.